sympy/core/containers.py

```diff
@@ -144,6 +145,33 @@ def index(self, value, start=None, stop=None):
         else:
             return self.args.index(value, start, stop)
 
+    @property
+    def kind(self):
+        """
+        Kind method is used to determine the type of Tuple objects.
+        Kind of Tuple will always be ``TupleKind`` with kinds of all arguments
+        as parameters
+
+        ``Tuple(Matrix([1, 2]), 1).kind`` for this expression kind will be ``TupleKind``,
+        ``MatrixKind(NumberKind), NumberKind`` as parameters because 1st argument is ``Matrix([1, 2])``
+        which have ``MatrixKind(NumberKind)`` as kind and 2nd argument is ``1`` which have
+        ``NumberKind`` as kind. So kind of this expression will be ``TupleKind(MatrixKind(NumberKind), NumberKind)``
```

Comment:
Use your editor to wrap long lines.

Reply:
This comment applies to all docstrings

Reply:
I should format this by adding newline and commit this?